src/main/java/com/spartronics4915/frc2019/Robot.java

```diff
@@ -138,6 +140,8 @@ public void disabledInit()
             Drive.getInstance().zeroSensors();
             RobotStateEstimator.getInstance().resetRobotStateMaps(Timer.getFPGATimestamp());
 
+            MySubsystem.getInstance(); // Not entirely sure if this would go here
```

Comment:
`getInstance` makes a `new` instance of your subsystem, if it doesn't already exist, stores that instance, and returns it. It's just like doing `new MySubsystem()`, except you can only ever have one instance of `MySubsystem`. The only reason you would do `getInstance` is because you actually want the instance of your subsystem, but you're not doing anything with it here, so this call to `getInstance` isn't needed.